src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -204,9 +204,13 @@ protected function dotize($id)
     /**
      * @internal
      */
-    protected function escape(string $string): string
+    protected function escape($value): string
```

Comment:
This should throw a type error if the subject is not a string neither implements a class with `__toString()`

Reply:
The `attribute` array is set from this file and can only contain `string` or `bool`.